docs/rfcs/004-issueish-list.md

```diff
@@ -0,0 +1,98 @@
+# Issueish List
+
+## Status
+
+Proposed
+
+## Summary
+
+Display a list of all open pull requests in the current repository in the GitHub tab.
+
+## Motivation
+
+To provide a navigational element that makes sense even if you aren't on an active feature branch.
+
+To give users a way to see an overview of what's going on in the repository.
+
+As an initial building block toward a pull request review workflow.
+
+## Explanation
+
+### Accordion Lists
+
+Within the GitHub panel, render a vertical stack of two collapsible lists of _issueish_ (pull request or issue) items:
+
+_First list: current pull request_. If the active branch is associated with one or more open pull requests on the GitHub repository, render an item for each.
+
+_Second list: all open pull requests_. List all open pull requests on the GitHub repository.
+
+Each list has a "collapse arrow" in its header. Clicking the collapse arrow toggles the visibility of that list's items, accordion-style.
+
+If either list exceeds 20 items, truncate the list and render a "More" link after its final item. Clicking "more" opens the corresponding search on GitHub.
+
+Each list item renders a tile containing a compact set of information about that pull request:
+
+* Mini author avatar
+* Title, truncated if necessary
+* Terse relative timestamp (1d, 2h, 30m)
+* Pull request state: open, closed, merged
+* Status check summary
+
+_TBD: Tile design_
+
+Clicking on a list item opens an issueish pane item for the chosen issueish in the same pane container as the parent component (by default, the right dock). If the issuish pane item is already open, it is activated instead.
+
+### Issueish Pane Item: Pull Request
+
+For a pull request, the issueish pane shows:
+
+* Author avatar
+* Title
+* Controls for actions:
+  * "Checkout" to fetch (if necessary) and check out the pull request. Only enabled if the current pull request is not the current one.
+  * "Browse" to launch a browser on the corresponding GitHub page.
+  * "Merge" to merge the pull request on GitHub if it is open.
+  * "Close" to close the pull request, unmerged, if it is open.
+  * "Re-open" to re-open a pull request if it is closed.
+* Pull request overview details: commit count, files changed.
+* Full description as rendered markdown.
```

Comment:
Since the full description could be pretty long, we could have max height with the option to "show more". Or have the full description at the "end" so you don't have to scroll down too much to see the status checks.

Reply:
> Could we make the description the bit that scrolls?

If we move the merge/close buttons to the bottom, having the description/comments scrollable, would be a good idea. 👍

An alternative to "Show more" for very long descriptions could be a "jump to new comments", in case we wanted to add some sort of read/new notification.